### PR TITLE
Refactor translations

### PR DESCRIPTION
**Pull request by jeroach**
[US121645](https://rally1.rallydev.com/#/detail/userstory/447878022404?fdp=true): Localization and Translations M1 work

Okay, this is a large PR, but there isn't really a lot of logic changes. Just a few types of changes, repeated all over the place. I'll give some overall context here, and call out to specific examples inline.

1. Update localization logic for `serge` and add new translation helper `localize-consistent-evaluation.js` to match `core` pattern.
2. Update all references to `LocalizeMixin` to `LocalizeConsistentEvaluation` (and remove unneeded `getLocalizeResources` method).
3. Update lang files to be `.js` files instead of `.json`
4. A few small fixes found along the way

### CODE REVIEW

File: components/consistent-evaluation-page.js [hunk 1]
```diff
@@ -15,12 +15,11 @@ import { Grade, GradeType } from '@brightspace-ui-labs/grade-result/src/controll
 import { Awaiter } from './awaiter.js';
 import { ConsistentEvaluationController } from './controllers/ConsistentEvaluationController.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
-import { loadLocalizationResources } from './locale.js';
-import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
+import { LocalizeConsistentEvaluation } from '../lang/localize-consistent-evaluation.js';
```
Comment (jeroach) [inline components/consistent-evaluation-page.js:18]: Example of step 2 - changing the `import` statement, and removing the old ones

File: components/consistent-evaluation-page.js [hunk 2]
```diff
@@ -15,12 +15,11 @@ import { Grade, GradeType } from '@brightspace-ui-labs/grade-result/src/controll
...
 
 const DIALOG_ACTION_LEAVE = 'leave';
 
-export default class ConsistentEvaluationPage extends LocalizeMixin(LitElement) {
+export default class ConsistentEvaluationPage extends LocalizeConsistentEvaluation(LitElement) {
```
Comment (jeroach) [inline components/consistent-evaluation-page.js:22]: Example of step 2 - changing the mixin

File: components/consistent-evaluation-page.js [hunk 3]
```diff
@@ -164,10 +163,6 @@ export default class ConsistentEvaluationPage extends LocalizeMixin(LitElement)
 		`;
 	}
 
-	static async getLocalizeResources(langs) {
```
Comment (jeroach) [inline components/consistent-evaluation-page.js:167]: Example of step 2 - remove unneeded method

File: components/right-panel/consistent-evaluation-grade-result.js [hunk 4]
```diff
@@ -120,9 +112,8 @@ export class ConsistentEvaluationGradeResult extends LocalizeMixin(LitElement) {
 		}
 
 		return html`
-			<d2l-consistent-evaluation-right-panel-block title="Overall Grade">
+			<d2l-consistent-evaluation-right-panel-block title="${this.localize('overallGrade')}">
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-grade-result.js:115]: Example of step 4 - this was not previously translated

File: components/right-panel/consistent-evaluation-grade-result.js [hunk 5]
```diff
@@ -27,16 +26,14 @@ export class ConsistentEvaluationGradeResult extends LocalizeMixin(LitElement) {
 				attribute:'custom-manual-override-clear-text',
 				type: String
 			},
-			labelText: {
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-grade-result.js:30]: Example of step 4 - `labelText` is not required

File: consistent-evaluation.serge.json [hunk 6]
```diff
@@ -1,20 +1,24 @@
...
-      "ko-kr ko",
-      "nl-nl nl",
-      "pt-br pt",
-      "sv-se sv",
-      "tr-tr tr",
-      "zh-cn zh",
-      "zh-tw zh-TW"
-    ]
+	"name": "consistentEvaluation",
+	"parser_plugin": {
```
Comment (jeroach) [inline consistent-evaluation.serge.json:3]: Step 1 - use `parser_plugin` and update filenames to use `.js` instead of `.json`

File: lang/ar.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
+/* eslint quotes: 0 */
```
Comment (jeroach) [inline lang/ar.js:1]: Example of step 3 - rename file to `.js` and tweak slightly to match

File: lang/en.js [hunk 8]
```diff
@@ -1,4 +1,8 @@
-{
+/* eslint quotes: 0 */
+
+export default {
+	"GB": " GB",
```
Comment (jeroach) [inline lang/en.js:4]: For the English file, I added the missing terms, and re-ordered all of the terms alphabetically.

Serge auto orders all of the translated files alphabetically, but we control the `en` file.

File: lang/localize-consistent-evaluation.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
+import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
+
+export const LocalizeConsistentEvaluation = superclass => class extends LocalizeMixin(superclass) {
```
Comment (jeroach) [inline lang/localize-consistent-evaluation.js:3]: Step 1 - this is the new helper that extends `LocalizeMixin` directly - this mirrors the pattern that UI/core uses.

**Review by JamieTran — APPROVED** [review]

**Review by vix633 — APPROVED** [review]

**Review by AlexBedley — APPROVED** [review]

**Pull request merged**